test/realizations/catchments/Bmi_C_Adapter_Test.cpp

```diff
@@ -202,7 +247,7 @@ TEST_F(Bmi_C_Adapter_Test, Update_0_h) {
 
 /** Test that Schaake Runoff output variable values can be retrieved. */
 TEST_F(Bmi_C_Adapter_Test, GetValue_0_a_0) {
-    int out_var_index = 0;
+    int out_var_index = 0;  //JG: out_var_index = 1; 0->RAIN_RATE
```

Comment:
reconcile this comment for test `GetValue_0_a_[ 0 - 4 ]`

Reply:
ok